wrappers/services.go

```diff
@@ -59,25 +59,55 @@ func generateSnapServiceFile(app *snap.AppInfo) (string, error) {
 }
 
 // StartSnapServices starts service units for the applications from the snap which are services.
-func StartSnapServices(s *snap.Info, inter interacter) error {
+func StartSnapServices(s *snap.Info, inter interacter) (err error) {
+	hasReloaded := false
+	sysd := systemd.New(dirs.GlobalRootDir, inter)
+	var enabled []*snap.AppInfo
+	var started []*snap.AppInfo
+
+	defer func() {
```

Comment:
You can if the helper is defined in the same function. You only need a tiny function that looks at `err` and calls a regular function like StopService or similar.